api/types/events/events.proto

```diff
@@ -1699,6 +1699,8 @@ message WindowsDesktopSessionEnd {
     ];
     // DesktopName is the name of the desktop resource.
     string DesktopName = 11 [ (gogoproto.jsontag) = "desktop_name" ];
+    // Recorded is true if the session was recorded, false otherwise.
+    bool Recorded = 12 [ (gogoproto.jsontag) = "recorded" ];
```

Comment:
Not sure why these are showing up as changes, since [they're already on `zmb3/desktop-session-recording`](https://github.com/gravitational/teleport/blob/zmb3/desktop-session-recording/api/types/events/events.proto#L1702-L1703)

Reply:
Probably because of a force push to the other branch. I'll stop doing that.